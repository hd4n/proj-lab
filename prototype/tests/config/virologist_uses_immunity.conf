#effekt_nev id duration
@effects
im i1 10

#agent_nev id effekt_id
@agents
va va1 i1

#kod_nev id agent_id aminocost nucleocost
@codes
co cd1 va1 0 0

#mezo_nev id [tartalom]
@fields
em e1

#id akt_mezo [item_id1;item_id2...] [material_id1;material_id2...] [effect_id1;effect_id2...]
@virologists
v1 e1 null null null

