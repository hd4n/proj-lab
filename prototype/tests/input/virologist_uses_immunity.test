load virologist_uses_immunity.conf
getstats v1
craft v1 cd1
useagent v1 v1 i1
getstats v1
