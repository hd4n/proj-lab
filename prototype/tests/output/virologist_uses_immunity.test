Load: sikeres

stats: v1
	fi_e1
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_null
craft: v1 cd1
useagent: v1 v1 va1

stats: v1
	fi_e1
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_0
	neededNucleo_0
	co_null
	ma_null
	ag_null
	eq_null
	eqa_null
	ef_i1
