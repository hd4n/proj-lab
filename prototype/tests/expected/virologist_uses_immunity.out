Load: sikeres

stats v1:
fi_e1
eq_null
ma_null
ef_null
co_null

craft: c1 cd1 va1 #kell?

useagent:
v1+v1
immunity

stats v1:
fi_e1
eq_null
ma_null
ef_i1
co_null